utility-belt/pom.xml

```diff
@@ -193,11 +199,11 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
-         <dependency>
-            <groupId>io.confluent</groupId>
-            <artifactId>confluent-log4j</artifactId>
-            <scope>compile</scope>
-         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-reload4j</artifactId>
+            <version>1.7.36</version>
+        </dependency>
```

Comment:
I don't think a version is supposed to be specified here, going off the previous updates to the future releases: https://github.com/confluentinc/common-docker/blob/master/utility-belt/pom.xml#L228